test/unit/observable.js

```diff
@@ -1,19 +1,19 @@
 QUnit.module('fabric.Observable');
 
-QUnit.test('fabric.Observable exists', function(assert) {
+QUnit.test('fabric.Observable exists', function (assert) {
   assert.ok(fabric.Observable);
-  assert.ok(fabric.Observable.fire);
-  assert.ok(fabric.Observable.on);
-  assert.ok(fabric.Observable.once);
-  assert.ok(fabric.Observable.off);
+  var o = new fabric.Observable();
```

Comment:
i undertstand this looks better, but is a mixin really supposed to to be initialized on its own?

Reply:
> my intention is to make CommonMethods extend Observable

Reply:
line node's `EventEmitter`